helpers_objectstorage.go

```diff
@@ -28,6 +30,11 @@ var bucketSchema = map[string]*schema.Schema{
 		Required: true,
 		Computed: false,
 	},
+	"access_type": {
+		Type:     schema.TypeString,
+		Computed: false,
+		Optional: true,
```

Comment:
oh, while we're at it we probably want to add something like `ValidationFunc: validation.StringInSlice([]string{baremetal.NoPublicAccess, baremetal.ObjectRead}, true)`, see https://godoc.org/github.com/hashicorp/terraform/helper/validation#StringInSlice